src/linearmixedmodel.jl

```diff
@@ -397,15 +397,18 @@ function fit!(m::LinearMixedModel{T}; verbose::Bool = false, REML::Bool = false)
     end
     opt = Opt(optsum)
     optsum.REML = REML
+    prog = ProgressUnknown(; showspeed=true)
     function obj(x, g)
         isempty(g) || throw(ArgumentError("g should be empty for this objective"))
         val = objective(updateL!(setθ!(m, x)))
+        ProgressMeter.next!(prog; showvalues = [(:objective, val),])
         verbose && println(round(val, digits = 5), " ", x)
```

Comment:
```suggestion
        verbose && ProgressMeter.next!(prog; showvalues = [(:objective, round(val, digits = 5)), (:theta, x)])
```

Reply:
I would prefer not to try to show `(:theta, x)` at each iteration because I think it would be too distracting.  For the example I suggested using the `:mrk17_exp1` data set, the length of `x` is 36, which makes for a lot of writing to the console and, I fear, a lot of distraction during the iterations.

I like  printing out the objective in that, even if it is changing too quickly to be able to read the numbers, you can see what parts of the objective have stabilized.  I prefer to show it in full (i.e. without the call to `round`) so that the stabilization doesn't continues to be visible even if the changes are past the 5th digit.

I agree that it would be best to preface the call to `next!` with `verbose &&`.  Should we do the same in the call to `ProgressUnknown` and `ProgressMeter.finish!`?

Shall we change the "Progress" annotation to something like "Minimizing" or "Optimizing parameters:"?

I can make similar changes in the `fit` method for GLMMs once we settle on things here.

Reply:
I completely agree with the problems of a big output and am happy with that proposal -- I was mostly just mimicking the old output. :smile: 

I have a vague preference for "Minimizing"  -- it's both shorter and more specific.

Reply:
"Minimizing" it is then.